public/static/pathways/her2_pathway.json

```
@@ -2,7 +2,7 @@
    "id": "tB4jsxTDL",
    "name": "Early Stage HER2+ (Draft)",
    "description": "Pathway for the Early Stage HER2+ Initial Medical Consult Breast Cancer Pathway. This is a draft pathway.",
-   "library": "",
+   "library": [""],
```

Should this just be an empty list instead?
```suggestion
   "library": [],
```
Same goes for neoadjuvant_pathway.json

---

Good call, changed.